models/post.model.js

```diff
@@ -0,0 +1,12 @@
+var mongoose = require( 'mongoose' );
+var Schema = mongoose.Schema;
+
+var PostSchema = new Schema({
+  postTime: Number,
+  username: String,
+  endTime: Number,
+  setup: Schema.Types.Mixed,
```

Comment:
In a future issue, well fix this so it only expects and Array - currently it sometimes receives a String